tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -229,6 +229,7 @@ private static function findTestFiles(): iterable
 		yield __DIR__ . '/../Rules/Methods/data/bug-4801.php';
 		yield __DIR__ . '/../Rules/Arrays/data/narrow-superglobal.php';
 		yield __DIR__ . '/../Rules/Methods/data/bug-12927.php';
+		yield __DIR__ . '/nsrt/bug-8886.php';
```

Comment:
as the test-file is in the `nsrt` folder, this line here is not necessary

Reply:
maybe leave a comment in this file here for the next contributor, to make people aware `nsrt/` containing files will be automatically picked up